user_docs/en/userGuide.t2t

```diff
@@ -1321,6 +1360,28 @@ This option won't be available if your braille display only supports automatic p
 
 You may consult the documentation for your braille display in the section [Supported Braille Displays #SupportedBrailleDisplays] to check for more details on the supported types of communication and available ports.
 
++++ Vision +++[VisionSettings]
+The Vision category in the NVDA Settings dialog allows you to enable, disable and configure [vision enhancement providers #Vision].
+This settings category contains the following options:
+
+==== Vision enhancement providers ====[VisionSettingsVisionEnhancementProviders]

```

Comment:
Ah, I hadn't realized that you didn't touch the userGuide yet. Do you want me to do this?

Reply:
I think we should get the rest of the interface sorted first. Then we can circle back to this.